### PR TITLE
Add sscocaller

### PR DESCRIPTION
**Pull request by ruqianl**
Add sscocaller recipe
----
 sscocaller is command line tool that analyzes single-cell DNA sequencing dataset of gamete cells for crossover detection.

Please read the [guidelines for Bioconda recipes](https://bioconda.github.io/contributor/guidelines.html) before opening a pull request (PR).

* If this PR adds or updates a recipe, use "Add" or "Update" appropriately as the first word in its title.
* New recipes not directly relevant to the biological sciences need to be submitted to the [conda-forge channel](https://conda-forge.org/docs/) instead of Bioconda.
* PRs require reviews prior to being merged. Once your PR is passing tests and ready to be merged, please issue the `@BiocondaBot please add label` command.
* Please post questions [on Gitter](https://gitter.im/bioconda/Lobby) or ping `@bioconda/core` in a comment.

<details>
  <summary>Please use the following BiocondaBot commands:</summary>

Everyone has access to the following BiocondaBot commands, which can be given in a comment:

<table>
  <tr>
    <td><code>@BiocondaBot please update</code></td>
    <td>Merge the master branch into a PR.</td>
  </tr>
  <tr>
    <td><code>@BiocondaBot please add label</code></td>
    <td>Add the <code>please review & merge</code> label.</td>
  </tr>
  <tr>
    <td><code>@BiocondaBot please fetch artifacts</code></td>
    <td>Post links to CI-built packages/containers. <br />You can use this to test packages locally.</td>
  </tr>
</table>

For members of the Bioconda project, the following command is also available:

<table>
  <tr>
    <td><code>@BiocondaBot please merge</code></td>
    <td>Upload built packages/containers and merge a PR. <br />Someone must approve a PR first! <br />This reduces CI build time by reusing built artifacts.</td>
  </tr>
</table>

Also, the bot watches for comments from non-members that include `@bioconda/<team>` and will automatically re-post them to notify the addressed `<team>`.

</details>

### CODE REVIEW

File: recipes/sscocaller/build.sh [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+chmod 775 configure
+sed -i.bak 's/exit(strncmp(ZLIB_VERSION, "1.2.5", 5) < 0);/exit(ZLIB_VERNUM < 0x1250);/g' configure
+./configure --with-readline=no --with-x=no --prefix=$PREFIX --enable-libcurl CFLAGS="-I$PREFIX/include" LDFLAGS="$LDFLAGS" CXXFLAGS="$CXXFLAGS" LIBICONV="$PREFIX/lib"
+cd src/nmath/standalone/
+make
+
+cp libRmath* "${PREFIX}/lib"
+
+cd ../../../../
+rm -r r-source
```
Comment (dpryan79) [inline recipes/sscocaller/build.sh:34]: Why is this bringing in an ancient version of R? If you need R, simply require r-base.

Comment (ruqianl) [reply recipes/sscocaller/build.sh:34]: Thanks, Devon. I need the R standalone library compiled which the nim package imports some functions from, if that makes sense.

Comment (dpryan79) [reply recipes/sscocaller/build.sh:34]: Right, I guess we don't package that with r-base at the moment.

**Comment by ruqianl** [comment]
@BiocondaBot please add label

**Review by dpryan79 — APPROVED** [review]

**Comment by dpryan79** [comment]
@bioconda-bot please merge

**Pull request merged**